src/models/AppState.js

```diff
@@ -602,6 +602,15 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
         fn: function (policy) {
           return (policy && policy.factorsPolicyInfo) ? policy.factorsPolicyInfo: null;
         }
+      },
+      'customFactorRedirectUrl': {
+        deps: ['lastAuthResponse'],
+        fn: function (res) {
+          if (!res._links || !res._links.next || !res._links.next.href) {
+            return null;
+          }
+          return res._links.next.href;
```

Comment:
we should check that the `name` of the link is `redirect`, so we are sure it is a url redirect and not any other `next` link

Reply:
👍